src/ansys/fluent/core/solver/settings_builtin_data.py

```diff
@@ -1170,9 +1170,36 @@
     "ReadCase": ("Command", "file.read_case"),
     "ReadData": ("Command", "file.read_data"),
     "ReadCaseData": ("Command", "file.read_case_data"),
-    "WriteCase": ("Command", "file.write_case"),
-    "WriteData": ("Command", "file.write_data"),
-    "WriteCaseData": ("Command", "file.write_case_data"),
+    "WriteCase": (
+        "Singleton",
+        {
+            FluentVersion.v261: "file.write_case",
```

Comment:
@mkundu1 

How would you feel about introducing a small DSL here, with helpers like `since()`, `until()`, `only_at()`, to declare version-specific paths more clearly?

I'm thinking this might be worth building as reusable infrastructure around `FluentVersion`. Version ranges could be helpful in other areas of the codebase too.

What do you think?

Reply:
Good idea. I've added somthing [here](https://github.com/ansys/pyfluent/pull/4172), will discuss further.

Reply:
LGTM! Thank you.

Reply:
Thanks, I'll update the current file to use those new functions in the other PR.